### PR TITLE
Fix wrong weight value of weighted targets

### PR DESCRIPTION
**Pull request by mpon**
*Issue #, if available:*

Fixed the request was not routed by weight, since the value of the weight is not specified as a percent.

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/apps/colorapp/servicemesh/delete.sh [hunk 1]
```diff
@@ -20,9 +20,16 @@ err() {
...
+    shift
+    for e; do [[ "$e" == "$match" ]] && return 0; done
+    return 1
+}
+
 sanity_check() {
-    if [ ! -n "${SUPPORTED_REGIONS[$AWS_DEFAULT_REGION]}" ]; then
-        err "Region ${AWS_DEFAULT_REGION} is not supported at this time (Supported regions: ${!SUPPORTED_REGIONS[*]})"
+    if ! contains "${AWS_DEFAULT_REGION}" "${SUPPORTED_REGIONS[@]}"; then
+        err "Region ${AWS_DEFAULT_REGION} is not supported at this time (Supported regions: ${SUPPORTED_REGIONS[*]})"
```
Comment (mpon) [inline examples/apps/colorapp/servicemesh/delete.sh]: Fixed like #51 at the same time.

**Comment by mpon** [comment]
I actually tried it with `colorteller-route-blue-80-red-20.json`, but did not route it. I will check again.

**Comment by mpon** [comment]
I tried it again, this didn't matter as you told!
However, I had to stop and run again the container of `colorgateway` ECS service to reflect traffic routing as route:thinking:
So, I misunderstood  because of the specifying weight.

**Comment by Y0Username** [comment]
Yes updating route doesn't take affect sometimes, this is a known issue #54. However, it is sporadic.

**Comment by kiranmeduri** [comment]
@mpon you may be seeing #54

**Comment by kiranmeduri** [comment]
@mpon can you revert the route-config file changes and we can then proceed with the delete.sh file changes. Thank you!

**Comment by mpon** [comment]
@kiranmeduri I reverted it. Thanks for letting me know the issue!

**Pull request merged**